_data/projects/project-catalog.yml

```diff
@@ -0,0 +1,8 @@
+name: Project Catalog
+desc: Project-catalog is a web app created using next js. A place where university students can upload thier project info , images , markdown file and other users may search for projects as a reference.
+site: https://nextjs-project-manager-phi.vercel.app
+tags:
+  nextjs , mantine-ui , google analytics 4, 
+upforgrabs:
+  name: Project Catalog
+  link: https://nextjs-project-manager-phi.vercel.app
```

Comment:
This should be a URL to an issue tracker where contributors can see tasks to help with.

Without this I am not able to accept this listing.